readme.md

```diff
@@ -255,6 +255,11 @@ Host *
 
 If you're running into other issues when using SSH, please consult [GitHub's support article](https://help.github.com/articles/connecting-to-github-with-ssh/).
 
+### Ignore strategy
+The [ignore strategy](https://docs.npmjs.com/files/package.json#files) either maintained in the `files`-property (`package.json`) or in the `.npmignore`-file should minify your packages.
```

Comment:
"minify" means something else. It's more about "reducing" the size of the package.